docs/development/how-to-release.md

```diff
@@ -36,6 +36,8 @@ Namespaced tags may also contain postfix to document which tool version is packa
 ## Fixes
 *
 
-## Incompatible Changes [Remove if none exist, document mitigation if they do]
+<!-- Remove if none exist, document mitigation if they do
```

Comment:
I would like to suggest to have the `Incompatible Changes` section _always_, even if there is no incompatible change.

* The format of the release notes looks the same always
* It makes a difference if the section is omitted, or if it is present with _no incompatible changes_ . Having this the reader can see: "hey, they thought about it". Of course it might happen that there are incompatible changes, even if we think everything is compatible, but that is another story.

Reply:
I agree with your reasoning, updated.